internal/domain/usecase/getURL.go

```diff
@@ -0,0 +1,20 @@
+package usecase
+
+import (
+	"errors"
+
+	"github.com/Taboon/urlshortner/internal/storage"
+)
+
+var errUnknownID = errors.New("отсутствует такой ID")
```

Comment:
CHANGE REQUEST: it should be in another package such as `error`

Reply:
done